src/components/SessionExpired.js

```diff
@@ -0,0 +1,46 @@
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/Errors/ErrorMessage';
+import Link from 'components/Link';
+
+import {ConfigContext} from '../Context';
+import {flagNoActiveSubmission} from '../data/submissions';
+import useSessionTimeout from '../hooks/useSessionTimeout';
+
+const SessionExpired = () => {
+  const config = useContext(ConfigContext);
+  const [expired] = useSessionTimeout();
```

Comment:
we should check if the `reset` callback returned by `useSessionTimeout` should be called here or not - likely you can test this by using the appointment flow:

1. start an appointment
2. let the session timeout
3. verify that we get redirected to the session expiry
4. click the "return" link going back to the start page
5. it should be possible to start a new appointment submission (and complete this)

there may be some internal state about expiry date/timers that now prevents this.